### ISSUE TITLE
Suggestion: Movie Texture support

### ISSUE THREAD

**Issue by rafcolson** [GoranM/bdx#774]
Hello

BDX will export and render the Blender's Texture if it's an *Image*, but not if it's a *Movie*. I think we could add support for that. Have to look into it first, though.

**Comment by SolarLune** [comment]
Yeah, movie file support would be nice to add in. It looks like it might have been [declared out of scope](http://gamedev.stackexchange.com/questions/30607/play-videos-with-libgdx) for LibGDX itself, but there are extensions to do this for us. [GDX-Video seems to be one](https://github.com/libgdx/gdx-video). There might be alternative methods available as well - perhaps something that can play videos that we could grab the texture of, for example. [This dude's comment on the above StackExchange link gives the idea](http://gamedev.stackexchange.com/a/112242/10471).

**Comment by rafcolson** [comment]
Thanks for the info.

**Comment by SolarLune** [comment]
Yo, I'm interested in this now, too. 

I have a little animated intro at the beginning of my game, and the sprite size exceeds 10000 pixels height-wise, which basically won't load for certain graphics cards, I think. So, a video's probably gonna be needed. I'll look into it.

**Comment by SolarLune** [comment]
Hmm, I guess it's to be expected, but it doesn't look like there's very many video reading solutions for Java period, much less some that support all of BDX's targets. So it might be best to just leave it to Gdx-video (assuming they get it there at some point). Any ideas of how this would be implemented?

In the meantime, I'll just use an alternative solution for my need (I'll just output each sprite to an individual image file and load them sequentially). So no rush is necessary on this.

**Comment by GoranM** [comment]
> the sprite size exceeds 10000 pixels height-wise, which basically won't load for certain graphics cards

Does it really need to be that large?

> Any ideas of how this would be implemented?

Like from the ground up? I have no idea.

I just delegate video playback to libraries that can actually decode the data, like ffmpeg.

**Comment by SolarLune** [comment]
> Does it really need to be that large?

Well, originally it was a sprite strip (because it was an animation I drew in Aseprite, a spriting tool). I just have a set of individual image files now that I load sequentially.
